packages/caver-ipfs/src/index.js

```diff
@@ -53,19 +53,55 @@ class IPFS {
     }
 
     /**
-     * adds a file to IPFS
+     * adds a file to IPFS.
+     * If the `data` parameter is a `Buffer` or `ArrayBuffer`, upload to IPFS directly without using `fs`.
+     * If the `data` parameter is a string, use `fs` to read the file.
+     * Since `fs` is a module that can only be used on the server side, if it is client-side code,
+     * it must read the file in advance and pass the file contents in the format of `ArrayBuffer`.
      *
-     * @param {string|Buffer} data The file path string or file contents.
+     * If you get a "Error: Can't resolve'fs'" error when building your client code, add the following to your "webpack.config.json" file.
+     * @example
+     * module.exports = {
+     *     ...
+     *     node: {
+     *         fs: 'empty',
+     *     },
+     *     ...
+     * }
+     * @returns {null}
+     *
+     * If you use next web framework, add the following to your "next.config.json" file.
```

Comment:
```suggestion
     * If you use next web framework(https://nextjs.org/), add the following to your "next.config.json" file.
```

Reply:
This is outdated, so i applied separately.